fuzzinator/call/gdb_backtrace_decorator.py

```diff
@@ -74,6 +78,7 @@ def filter(*args, **kwargs):
                     child.terminate(force=True)
                     issue['backtrace'] = backtrace
                 except Exception:
+                    logger.debug('Failed to obtain gdb backtrace')
                     pass
```

Comment:
Since the `except` block is non-empty after adding the log message, we can remove this no-op `pass` statement.

Reply:
done